packages/google-cloud-serverless/src/gcpfunction/http.ts

```diff
@@ -44,6 +52,9 @@ function _wrapHttpFunction(fn: HttpFunction, options: Partial<WrapperOptions>):
     const baggage = req.headers?.baggage;
 
     return continueTrace({ sentryTrace, baggage }, () => {
+      const normalizedRequest = httpRequestToRequestEventData(req);
```

Comment:
IMHO it is better to set this before we start the span, so we can 100% ensure this is already set for the span.